.vscode/launch.json

```diff
@@ -6,25 +6,42 @@
             "type": "go",
             "request": "launch",
             "mode": "test",
-            "program": "${workspaceFolder}/tests/e2e",
+            "program": "${workspaceFolder}/x/concentrated-liquidity",
             "args": [
                 "-test.timeout",
                 "30m",
                 "-test.run",
-                "IntegrationTestSuite",
+                "TestKeeperTestSuite/TestWithdrawPosition",
                 "-test.v"
             ],
-            "buildFlags": "-tags e2e",
-            "env": {
-                "OSMOSIS_E2E": "True",
-                "OSMOSIS_E2E_SKIP_IBC": "true",
-                "OSMOSIS_E2E_SKIP_UPGRADE": "true",
-                "OSMOSIS_E2E_SKIP_CLEANUP": "true",
-                "OSMOSIS_E2E_SKIP_STATE_SYNC": "true",
-                "OSMOSIS_E2E_UPGRADE_VERSION": "v16",
-                "OSMOSIS_E2E_DEBUG_LOG": "false",
-            },
-            "preLaunchTask": "e2e-setup"
-        }
+        },
+        {
+            "name": "inverse",
+            "type": "go",
+            "request": "launch",
+            "mode": "test",
+            "program": "${workspaceFolder}/x/concentrated-liquidity",
+            "args": [
+                "-test.timeout",
+                "30m",
+                "-test.run",
+                "TestKeeperTestSuite/TestInverseRelation_CreatePosition_WithdrawPosition/lower_tick",
+                "-test.v"
+            ],
+        },
+        {
+            "name": "TestUpdatePosition",
+            "type": "go",
+            "request": "launch",
+            "mode": "test",
+            "program": "${workspaceFolder}/x/concentrated-liquidity",
```

Comment:
I assume these are left over from running the debugger?

Reply:
Yeah, I'm keeping this until all comments are addressed to help with debugging if any major issues arise

Reply:
Its feels ridiculous that we need to add these for each individual test to use the debugger... surely there is some other way / authomation :(